04-HTML5-APIs/Ex02/script.js

```diff
@@ -0,0 +1,34 @@
+function dragOver(e) {
+	e.preventDefault();
+	e.stopPropagation();
+	e.dataTransfer.dropEffect = 'copy';
+}
+function dropFiles(e) {
+	e.preventDefault();
+	e.stopPropagation();
+	let files = e.dataTransfer.files;
+	var file = files[0];
+	if (file.type == 'text/plain') {
+		function readFile(f) {
+			let reader = new FileReader();
+			reader.readAsText(f);
+			reader.onload = function() {
+				let newText = reader.result;
+				document.getElementById('dropzone').textContent = newText;
+			}
+		}
+	readFile(file);
+	} else if (file.type.match('image/*')) {
```

Comment:
This step is not need, but good to have it! The exercise description ask only for read text. +1